validation-test/StdlibUnittest/ChildProcessShutdown/RequireOwnProcess.swift

```diff
@@ -0,0 +1,42 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+
+import StdlibUnittest
+#if os(macOS) || os(iOS) || os(tvOS) || os(watchOS)
+  import Darwin
+#elseif os(Linux) || os(FreeBSD) || os(PS4) || os(Android) || os(Cygwin) || os(Haiku)
+  import Glibc
+#elseif os(Windows)
+  import MSVCRT
+#else
+#error("Unsupported platform")
+#endif
+
+//
+// Test that a test runs in its own child process if asked.
+//
+
+enum Globals {
+  static var globalVariable = 0
+}
+
+var TestSuiteRequireNewProcess = TestSuite("TestSuiteRequireNewProcess")
+
+TestSuiteRequireNewProcess.test("RequireOwnProcessBefore")
+  .requireOwnProcess()
```

Comment:
I meant *not* having this one marked `requireOwnProcess`, to test that your "next" checking works.